app/upgrades.go

```diff
@@ -165,3 +183,17 @@ func UpgradeICA(ctx sdk.Context, app *App, versionMap *module.VersionMap) {
 	icamodule.InitModule(ctx, controllerParams, hostParams)
 	app.Logger().Info("Finished initializing ICA")
 }
+
+func RemoveLeaveGroupMsgFee(ctx sdk.Context, app *App) error {
+	typeURL := sdk.MsgTypeURL(&group.MsgLeaveGroup{})
+	app.Logger().Info("deleting MsgLeaveGroup msg fee if it exists")
+	err := app.MsgFeesKeeper.RemoveMsgFee(ctx, typeURL)
+	if err == nil {
+		return nil
+	}
+	if errors.Is(err, msgfeestypes.ErrMsgFeeDoesNotExist) {
+		return nil
+	}
+	app.Logger().Error("error deleting MsgLeaveGroup msg fee", "error", err)
+	return err
+}
```

Comment:
This might not be needed anymore. Waiting a bit to find out.

Reply:
What's your take on this now @SpicyLemon ?